test/e2e/framework/util.go

```diff
@@ -428,6 +430,11 @@ func getDefaultClusterIPFamily(c clientset.Interface) string {
 	return "ipv4"
 }
 
+// ClusterIsIPv6 returns true if the cluster is IPv6
+func ClusterIsIPv6() bool {
+	return TestContext.IPFamily == "ipv6"
+}
+
```

Comment:
One could also argue that this function doesn't belong into `util.go`. As it works on `TestContext`, wouldn't it be better to put this into `test_context.go`? Perhaps even as a method?

```
func (tc TestContextType) ClusterIsIPv6() bool {
    return tc.IPFamily == "ipv6"
}
```

I'm not asking for that change, just wanted to point it out...

Reply:
I really think you are right, will submit a PR with that change once this is merged if you don't mind.

Reply:
will submit as a follow up once this and your PR disabling the unit test is merged 
https://github.com/kubernetes/kubernetes/commit/45d6f08868ce2729182aae5734a00e5e27ae08f9

Reply:
Sounds good. :+1: